### ISSUE TITLE
useNuxtApp() don't work.

### ISSUE THREAD

**Issue by cwe-dev** [hirotaka/storybook-addon-nuxt#27]
This addon does not solve this problem? I reproduced it

https://github.com/storybookjs/storybook/issues/21196

**Comment by prnews-io-tech** [comment]
No, it just replaces it with an empty stub: https://github.com/hirotaka/storybook-addon-nuxt/blob/main/runtime/composables.mjs
